samples/algorithms/PhaseFlip.qs

```diff
@@ -0,0 +1,160 @@
+/// # Sample
+/// Phase-Flip
+///
+/// # Description
+/// This sample demonstrates the three-qubit phase-flip code. This code is a
+/// simple quantum error correction strategy for protecting against a single
+/// phase-flip error by encoding a logical qubit into three physical qubits. A
+/// single phase-flip error is when one of the three physical qubits has its
+/// state changed erroneously in a way that is equivalent to applying the Z
+/// gate to it.
+///
+/// The phase-flip correction code works by checking the parity of the physical
+/// qubits. By measuring only their parity, the quantum superposition of the
+/// qubits is preserved the qubits. Because all the physical qubits are
+/// supposed to have the same state, when the parity checks detect a
+/// difference in state, the erroneous qubit can be identified and corrected.
+///
+/// This Q# program prepares a logical qubit encoded as three physical qubits
+/// with one of the qubits being phase-flipped. It then identifies and corrects
+/// the flipped qubit.
+namespace Sample {
+    open Microsoft.Quantum.Math;
+    open Microsoft.Quantum.Random;
+    open Microsoft.Quantum.Arrays;
+    open Microsoft.Quantum.Diagnostics;
+    open Microsoft.Quantum.Measurement;
+
+    @EntryPoint()
+    operation Main() : Result {
```

Comment:
Similar to my comment on the bit-flip code sample, I would suggest using a single 3-qubit register throughout `Main`, and make `EncodeAsLogicalQubit` receive a qubit register rather than two separate arguments.